src/main/resources/templates/add-edit.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml" xmlns:th="http://www.thymeleaf.org">
+<head>
+  <meta charset="UTF-8">
+  <title>Amazin Bookstore</title>
+
+  <!-- Use Tailwind for CSS -->
+  <script src="https://cdn.tailwindcss.com"></script>
+</head>
+<body>
+  <div>
+    <form th:unless="${book == null}" action="/home/edit-book" method="POST">
+      <p>ISBN: <input type="text" name="isbn" class="border-2 rounded-md" th:value="${book.getIsbn()}" disabled/>
+      <p>Title: <input type="text" name="title" class="border-2 rounded-md" th:value="${book.getTitle()}" disabled/>
+      <p>Description: <input type="text" name="description" class="border-2 rounded-md"/>
+      <p>Author: <input type="text" name="author" class="border-2 rounded-md" th:value="${book.getAuthor()}" disabled/>
+      <p>Publisher: <input type="text" name="publisher" class="border-2 rounded-md" th:value="${book.getPublisher()}" disabled/>
+      <p>Inventory: <input type="number" name="inventory" class="border-2 rounded-md"/>
```

Comment:
You can set the number to be >0 with adding `min=0`

Reply:
Thanks for the suggestion